src/SFA.DAS.AANHub.Api.UnitTests/Controllers/ApprenticesControllerTests.cs

```diff
@@ -1,80 +1,178 @@
 ﻿using AutoFixture.NUnit3;
 using FluentAssertions;
+using FluentValidation.Results;
 using MediatR;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Moq;
 using NUnit.Framework;
-
 using SFA.DAS.AANHub.Api.Controllers;
 using SFA.DAS.AANHub.Api.Models;
 using SFA.DAS.AANHub.Application.Apprentices.Commands;
 using SFA.DAS.AANHub.Application.Apprentices.Queries;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
 using SFA.DAS.AANHub.Application.UnitTests;
-
 using static SFA.DAS.AANHub.Domain.Common.Constants;
 
 namespace SFA.DAS.AANHub.Api.UnitTests.Controllers
 {
     public class ApprenticesControllerTests
     {
-        [Test, AutoMoqData]
+        [Test]
+        [AutoMoqData]
         public async Task CreateApprentice_InvokesRequest(
             [Frozen] Mock<IMediator> mediatorMock,
             [Greedy] ApprenticesController sut,
-            CreateApprenticeModel model, CreateApprenticeMemberCommand command, Guid userId, long apprenticeId)
+            CreateApprenticeModel model, CreateApprenticeMemberCommand command)
         {
-            var response = new CreateApprenticeMemberCommandResponse
+            var response = new ValidatedResponse<CreateApprenticeMemberCommandResponse>(new CreateApprenticeMemberCommandResponse
             {
                 MemberId = command.Id,
                 Status = MembershipStatus.Live
-            };
+            });
+
+
+            mediatorMock.Setup(m => m.Send(It.IsAny<CreateApprenticeMemberCommand>(),
+                It.IsAny<CancellationToken>())).ReturnsAsync(response);
 
-            mediatorMock.Setup(m => m.Send(It.Is<CreateApprenticeMemberCommand>(c => c.RequestedByMemberId == userId && c.ApprenticeId == apprenticeId), It.IsAny<CancellationToken>())).ReturnsAsync(response);
             var result = await sut.CreateApprentice(Guid.NewGuid(), model);
 
             result.As<CreatedAtActionResult>().ControllerName.Should().Be("Apprentices");
             result.As<CreatedAtActionResult>().ActionName.Should().Be("GetApprentice");
             result.As<CreatedAtActionResult>().StatusCode.Should().Be(StatusCodes.Status201Created);
```

Comment:
Add assertion on the Id in the response if possible

Reply:
Done